src/components/LocaleSelect.astro

```diff
@@ -0,0 +1,27 @@
+---
+import Select from 'fulldev-ui/components/Select.astro'
```

Comment:
destructure the props needed, and also a restprops

Reply:
There are no props needed